Formula/twilio@2.36.1.rb

```diff
@@ -0,0 +1,21 @@
+require "language/node"
+
+class TwilioAT2361 < Formula
+  desc "unleash the power of Twilio from your command prompt"
+  homepage "https://github.com/twilio/twilio-cli"
+  url "https://twilio-cli-prod.s3.amazonaws.com/twilio-v2.36.0/twilio-v2.36.0.tar.gz"
+  version "2.36.0"
+  sha256 "d896744ab6518e0aaab4cb0ce069a2c9185b9732716b0cd4de0725bb011c5194"
```

Comment:
mismatch in `version` and `sha256`

Title says version `2.36.1` but this'd install 2.36.0

Reply:
```
  url "https://twilio-cli-prod.s3.amazonaws.com/twilio-v2.36.1/twilio-v2.36.1.tar.gz"
  version "2.36.1"
  sha256 "9c57ba33583ebf7c3c2ca20100554a9c2050c6a707a51bdae71fac18e64c0e21"

```

Reply:
+1, [commit](https://github.com/twilio/homebrew-brew/commit/3a5bc4aabe0fc1837a2b36a08894c5490715c910)

Reply:
Fixed this, thanks